utils/load_economy.lua

```diff
@@ -27,13 +27,16 @@ function place_flags(eco, plr)
 end
 
 function place_roads(eco, plr)
-   for i = 1, #eco.roads do
-      local command = string.format("road = plr:place_road(map:get_field(%i,%i).immovable, %s ,true)", eco.roads[i].x, eco.roads[i].y, eco.roads[i].dirs)
+   -- Second carrier can't be placed currently, so we just fake a carrier
```

Comment:
did you try whether this si still valid?     
according to the actual documentation we can readouit the road_type, which wasn't possible when I converted the script from ancient beginning of Atlantean02.     
So we might simply readout the type and try to set the type and the workers that have been read.     
Otherwise we might readout the type and fake a second carrier for each busy road in the HQ.

Reply:
just tested it we can set them and so we might alter the script.   
By doing so i found an issue in Atlantean 02 as well

Reply:
easiest way would be to read the road type and use road:set_workers(road.valid_workers) with the risk of overdoing some second carriers which hadn't been available yet while the road was already busy

Reply:
I tried to test it, but it looks like I did something wrong, because I got errors when I tried to add second carriers. Now I was able to do it.

Some debugging showed though that sometimes more than the allowed carriers are dumped. I guess these are on their way to somewhere else? Anyway, I decided to keep it simple and use `valid_workers` instead as you suggested.